tidb-server/main.go

```diff
@@ -606,6 +606,8 @@ func setupLog() {
 	err := logutil.InitLogger(cfg.Log.ToLogConfig())
 	terror.MustNil(err)
 
+	logutil.InitGeneralQueryLog()
```

Comment:
I think the namespace is not reasonable. General log is not something that is a util. It is a business logic.

Reply:
Do you have suggestions about the code placement? This may also be related to dependency problems discussed above.